providers/flagd/src/test/resources/flagConfigurations/valid-long.json

```diff
@@ -82,16 +83,7 @@
             ]
           },
           "second",
-          {
-            "in": [
-              "Chrome",
-              {
-                "var": "userAgent"
-              }
-            ]
-          },
-          "third",
-          null
```

Comment:
This was actually invalid.

Reply:
Ohh, we need to update flagd example herer -  https://github.com/open-feature/flagd/blob/main/config/samples/example_flags.flagd.json

I burrowed it 😞